indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerConfig.java

```diff
@@ -171,6 +174,28 @@ public static HadoopDruidIndexerConfig fromString(String str)
     }
   }
 
+  @SuppressWarnings("unchecked")
+  public static HadoopDruidIndexerConfig fromDistributedFileSystem(String path)
+  {
+    try
+    {
+      Path pt = new Path(path);
+      FileSystem fs = FileSystem.get(pt.toUri(), new Configuration());
```

Comment:
this is probably going to break with paths that are not valid URIs either. We had a similar issue here https://github.com/druid-io/druid/pull/1454/files#diff-2f6961baa5178a54edc5289685bbcb0fR521